src/elements/content-uploader/__tests__/ContentUploader-test.js

```diff
@@ -81,26 +84,47 @@ describe('elements/content-uploader/ContentUploader', () => {
     });
 
     describe('removeFileFromUploadQueue()', () => {
-        test('should cancel and remove item from uploading queue', () => {
-            const wrapper = getWrapper();
-            const item = {
-                api: {
-                    cancel: jest.fn(),
-                },
-                status: STATUS_IN_PROGRESS,
-            };
+        const item = {
+            api: {
+                cancel: jest.fn(),
+            },
+            status: STATUS_IN_PROGRESS,
+        };
+        let wrapper;
+        let instance;
+        beforeEach(() => {
+            wrapper = getWrapper();
             wrapper.setState({
                 items: [item],
             });
-            const instance = wrapper.instance();
-            instance.upload = jest.fn();
+            instance = wrapper.instance();
+        });
 
+        test('should cancel and remove item from uploading queue', () => {
             instance.removeFileFromUploadQueue(item);
 
             expect(item.api.cancel).toBeCalled();
             expect(wrapper.state().items.length).toBe(0);
-            expect(instance.upload).toBeCalled();
         });
+
+        test.each([VIEW_ERROR, VIEW_UPLOAD_EMPTY, VIEW_UPLOAD_IN_PROGRESS, VIEW_UPLOAD_SUCCESS])(
+            'should continue uploading if other files are being uploaded',
```

Comment:
Can you use interpolation here to make the test cases unique while still being generated from `test.each`